### PR TITLE
Remove unused ref return error codes

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@vsadov, @dotnet/roslyn-compiler please review.

File: src/Compilers/CSharp/Portable/Errors/ErrorCode.cs [hunk 1]
```diff
@@ -1399,7 +1399,7 @@ internal enum ErrorCode
         ERR_RefReturnNonreturnableLocal = 8158,
         ERR_RefReturnNonreturnableLocal2 = 8159,
         ERR_RefReturnRangeVariable = 8160,
-        ERR_RefReturnRangeVariable2 = 8170,
+        // Available  = 8161-8170
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/Errors/ErrorCode.cs]: I didn't realize I had left a gap of 10 here when compacting. Consider compacting more if you want.

**Comment by jcouv** [comment]
:shipit:

**Comment by AlekseyTs** [comment]
:shipit:

File: src/Compilers/CSharp/Portable/Errors/ErrorCode.cs (hunk 1 above)
Comment (cston) [inline src/Compilers/CSharp/Portable/Errors/ErrorCode.cs]: Compacted all ref return error codes.

**Comment by VSadov** [comment]
LGTM

**Pull request merged**